scene/main/window.cpp

```diff
@@ -1435,7 +1435,7 @@ void Window::_notification(int p_what) {
 
 			tr_title = atr(title);
 #ifdef DEBUG_ENABLED
-			if (window_id == DisplayServer::MAIN_WINDOW_ID) {
+			if (window_id == DisplayServer::MAIN_WINDOW_ID && !Engine::get_singleton()->is_project_manager_hint()) {
 				// Append a suffix to the window title to denote that the project is running
 				// from a debug build (including the editor). Since this results in lower performance,
```

Comment:
I'm not sure if this comment needs to be changed.

Reply:
I can add a word on the project manager in the comment